examples/howto/plot_connectivity.py

```diff
@@ -81,7 +80,7 @@ def get_network(probability=1.0):
     for target in ['L5_pyramidal', 'L2_basket']:
         net.add_connection(src, target, location, receptor,
                            delay, weight, lamtha, probability=probability,
-			   conn_seed=conn_seed)
+                           conn_seed=conn_seed)
```

Comment:
Not sure what was going on here, but the indent was all messed up in my vscode editor.